benchmarks/trie/lookup_bench.cu

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <defaults.hpp>
+#include <utils.hpp>
+
+#include "../../tests/trie/trie_utils.hpp"
```

Comment:
You can use the system path here, i.e., `<cuco/tests/…>`.

Reply:
`<cuco/tests/..>` does not compile.  Using `<../tests/>` in recent commit.

Should we change top-level CMakeLists or benchmarks/CMakeLists to access `<cuco/tests/..>` from benchmarks?